install_homebrew.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+# A simple Homebrew installation wrapper, to hide the platform-specific stanzas:
+# macOS: https://brew.sh/
+# Linux: https://docs.brew.sh/Homebrew-on-Linux#install
+
+set -e
+umask 022
+
+if [ -z "$USER" ]; then
+  # Note: The Homebrew installation scripts both expect the USER environment
+  # variable to be set, so just check upfront
+  # https://github.com/Linuxbrew/install/issues/48
+  # https://github.com/Linuxbrew/install/pull/49
+  echo "Aborting install! No USER environment variable set"
+  exit 1;
+fi
+
+if [ $(id -u) = 0 ]; then
+  echo "Aborting install! Must not be run as root"
+  exit 2
+fi
+
+if command -v brew >/dev/null 2>&1; then
+  echo "Homebrew already installed 👍"
+  exit
+fi
+
+case $(uname) in
+  Linux)
+    if command -v curl >/dev/null 2>&1; then
+      echo "Installing Homebrew"
+      sh -c "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install.sh)"
```

Comment:
Expand out the short arguments for documentation purposes (if they have long equivalent). I guess you could reasonably extract a function so that you don't have to repeat it in the Darwin case, too